WebRoot/css/zm.css

```diff
@@ -4398,18 +4398,21 @@ HTML>BODY .appt-selected .appt_allday_body
 
 .ZmShortcutList {
 	padding: 1rem;
-	-moz-columns: 3;
-	-webkit-columns: 3;
-	columns: 3;
+	-moz-columns: 2;
```

Comment:
I am not very sure going that way. There doesn't seem to be a block for webkit , just GECKO, SAFAR & default I guess.